src/Blocks/SettingsBlocks.php

```diff
@@ -77,14 +78,25 @@ class SettingsBlocks implements UtilsSettingGlobalInterface, UtilsSettingInterfa
 	 */
 	protected GeolocationInterface $geolocation;
 
+	/**
+	 * Instance variable for manifest cache.
+	 *
+	 * @var ManifestCacheInterface
+	 */
+	protected $manifestCache;
```

Comment:
Why `protected` and not `private`? Is the class being extended so that the `$manifestCache` is shared with the child classes?

Reply:
I will change this to private, no problem